setup.py

```diff
@@ -76,11 +77,21 @@ def symlink_package(name: str, path: Path) -> None:
     cwd = Path(__file__).parent
     path_from = cwd / path
     path_to = os.path.join(cwd, *name.split("."))
+
+    # OSError: [WinError 1314] A required privilege is not held by the client
+    # Windows requires special permission to symlink. Fallback to copy
+    use_symlink = platform.system() != "Windows"
```

Comment:
nit: in [torchvision we used `os.name == 'nt'` to figure out if we were on Windows or not](https://github.com/pytorch/vision/blob/0dceac025615a1c2df6ec1675d8f9d7757432a49/torchvision/extension.py#L97)